src/workerd/io/worker.c++

```diff
@@ -599,40 +596,12 @@ struct Worker::Isolate::Impl {
     KJ_DISALLOW_COPY_AND_MOVE(Lock);
 
     void setupContext(v8::Local<v8::Context> context) {
-      // Set WebAssembly.Module @@HasInstance
-      setWebAssemblyModuleHasInstance(*lock, context);
-
       // The V8Inspector implements the `console` object.
       KJ_IF_SOME(i, impl.inspector) {
```

Comment:
Only real change is that Set WebAssembly.Module @@HasInstance is now only set after alerting the inspector of a new context.
This shouldn't have any actual effect in practice.